example_configs/thelounge.md

```diff
@@ -5,12 +5,19 @@
 For simple user auth with LLDAP on localhost adapt this in the main config.js:
 
 ```
-ldap: {
-  enable: true,
-  url: "ldap://127.0.0.1:3890",
-  tlsOptions: {},
-  primaryKey: "uid",
-  baseDN : "ou=people,dc=example,dc=com",
+      ldap: {
+        enable: true,
+        url: "ldap://localhost:389",
+        tlsOptions: {},
+        primaryKey: "uid",
+        searchDN: {
+            rootDN: "uid=ldap-editor,ou=people,DC=example,DC=com",
```

Comment:
What's the root DN? And why ldap-editor? Maybe we need to describe this a bit more

Reply:
rootDN/rootPassword are the credentials used to login to lldap and perform queries. (Similar to bind user and bind user password. the lounge uses a different name for it) 

`ldap-editor` user is a member of `lldap_strict_readonly` and `lldap_password_manager` groups.

Reply:
Right, this would be great to put in the documentation :)